.claude/commands/jira-task.md

```
@@ -0,0 +1,173 @@
+
+```bash
+/jira-task {ISSUE-KEY}
+/jira-task MEMBER-123
+/jira-task ORDER-456
+```
+
+---
+
+## 실행 프로세스
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)